massa-protocol-worker/src/checked_operations.rs

```diff
@@ -21,7 +21,7 @@ impl CheckedOperations {
         self.0.insert(prefix, *id).is_none()
     }
 
-    /// Get a set of [OperationIds] matching with the givec `prefix`.
+    /// Get a set of [Set<OperationId>] matching with the givec `prefix`.
```

Comment:
To me it looks more like the function returns an Option with just a single operation ID